Makefile

```diff
@@ -41,7 +41,7 @@ test: go_protos
 	@go test -v ./...
 
 vet: go_protos
-	@go vet -composites=false ./...
```

Comment:
By default this is true, just remove the flag.

Reply:
Will do :)